README.md

```diff
@@ -189,13 +190,12 @@ include the JIRA number associated with your JIRA work item in the commit
 message title.
 
 To report a new issue, open a ticket in the Hyperledger Fabric [JIRA](https://jira.hyperledger.org/projects/FAB/issues)
-and add `fabric-chaincode-evm` in the component field.
+under the Fabric Chaincode EVM (FABCE) project.
 
 ## Current Dependencies
 - Hyperledger Fabric [v1.4](https://github.com/hyperledger/fabric/releases/tag/v1.4.0). EVMCC can be run on Fabric 1.0 and newer.
 - Hyperledger Fabric SDK Go [revision = "beccd9cb1450fddfe426616e151d709c99f7ccdd"](https://github.com/hyperledger/fabric-sdk-go/tree/beccd9cb1450fddfe426616e151d709c99f7ccdd)
```

Comment:
The SDK Go was updated in the previous release notes, but was missed in the docs, since you are editing the file this needs to be updated as well. But this isn't a change in the current release
```suggestion
 - Hyperledger Fabric Go SDK [1.0.0-alpha5](https://github.com/hyperledger/fabric-sdk-go/releases/tag/v1.0.0-alpha5)
```